components/layout/GithubStargazer.client.vue

```diff
@@ -3,16 +3,23 @@
     <span v-else class="placeholder" style="width: 39px"></span>
 </template>
 
-<script>
-    import axios from "axios";
+<script setup>
+import { ref, onMounted } from 'vue';
+import axios from "axios";
 
-    export default {
-        data: () => ({
-            stargazersText: undefined
-        }),
-        async mounted() {
-            const response = await axios.get('/api/github')
-            this.stargazersText = Intl.NumberFormat('en-US').format(response.data.stargazers);
-        },
-    };
-</script>
+const stargazersText = ref(null);
+
+const formatStargazers = (count) => {
```

Comment:
What is wrong with using `Intl.NumberFormat()`?
Does it do something different?

Get inspiration from https://github.com/kestra-io/docs/blob/main/components/home/OpenSource.vue#L60C84-L63C5